parts/nav-bftop.php

```diff
@@ -11,8 +11,20 @@
 				<?php get_template_part( 'parts/nav', 'bfsubmenu' ); ?>
             </div>
         </div>
-        <div class="cell" style="width: 54px; height: 54px;"> 
-          User
+        <div class="cell" style="width: 54px; height: 54px;">
+        <a href="<?php bp_loggedin_user_link(); ?>" data-toggle="user-dropdown"><?php bp_loggedin_user_avatar( 'type=full' ); ?></a>
```

Comment:
One small detail: everything inside the div above should be indented one stop. That helps keep things organized.